docs/data/charts/gauge/gauge.md

```diff
@@ -80,9 +80,9 @@ import { Gauge } from '@mui/x-charts/Gauge';
 
 The second option is to make use of the following elements that are available within the Gauge module:
 
-- GaugeReferenceArc
-- GaugeValueArc
-- GaugeValueText
```

Comment:
I'm not sure about those. I kind of remember @samuelsycamore ask to use less those `<Component />` when it's clear we are talking about some components you can import

Reply:
As per [Sam's writing style guide](https://www.notion.so/mui-org/Style-conventions-for-MUI-components-and-products-dff7d2d7d1ba4a4abd47f48cf345b800?pvs=4#8f86b8e3c0fb49e1b4df3276efb3848e), I believe it's either wrapped in backticks with `< />` or "Gauge Reference Arc", spaced out. As we were specifically referring to the other modules available in the package, thought the backtics wrap was more appropriate 🤷 He'll be able to tell better, either way! :)